docs/userguide.md

```diff
@@ -592,8 +593,8 @@ behaviour is unacceptable.
 
 The RSP can be considered as more user friendly mechanism to distribute the
 replicas, where the inputs needed from the user at federated control plane are
-reduced. The user only needs to create the RSP resource and the mapping template
-resource, to distribute the replicas. It can also be considered as a more
+reduced. The user only needs to create the RSP resource and associated federated
+resource to distribute the replicas. It can also be considered as a more
```

Comment:
s/create the RSP resource and associated federated resource to distribute the replicas/
create the RSP resource and associated federated resource with only `spec.template` field filled in to distribute the replicas./

Reply:
Done